src/App.scss

```diff
@@ -1,3 +1,3 @@
-iframe {
-  display: none;
-}
+iframe {

```

Comment:
i don't see any reason for this styling, since there is no iframe on the document.

Reply:
hello Julian,

It already comes with code that I didn't create.

Reply:
oh, no problem then =)